File: demos/js/simple.js
#!/usr/bin/env gjs

const Gtk = imports.gi.Gtk;
const Abc = imports.gi.Abc;

Gtk.init(null, 0);

let w = new Gtk.Window()
let cb = new Abc.ColorBox()
cb.set_point(new Abc.Point({x : 5, y: 17}))
w.add(cb)
w.show_all()
w.connect ("destroy", function(){Gtk.main_quit()});
Gtk.main()

File: demos/python/simple.py
from gi.repository import Gtk, Abc, GLib

w = Gtk.Window()
w.connect("delete-event", Gtk.main_quit)
cb = Abc.ColorBox()
cb.set_point(Abc.Point(x=5,y=17))
w.add(cb)
w.show_all()
Gtk.main()
